pages/availability/index.tsx

```diff
@@ -344,14 +338,17 @@ export default function Availability(props) {
                         </div>
                     </div>
                 }
-                <Modal heading="Start and end times changed" description="The start and end times for your day have been changed successfully." open={successModalOpen} handleClose={closeSuccessModal} />
+                <Modal heading="Start and end times changed" description="The start and end times for your day have been changed successfully." open={successModalOpen} handleClose={() => setSuccessModalOpen(false)} />
```

Comment:
⛏️ I think that for the sake of consistency with the other components, we ought to keep `closeSuccessModal` and have that function `setSuccessModalOpen(false)`, since we do the same for the two other instances where we use a modal.

Reply:
The two other instances are those toggle* functions which I think may lead to unintentional behaviour (what if something toggles elsewhere and then toggles again somewhere else). I think it's better to explicitly set those values as well - which would solve the consistency thing but the other way around; I suppose.

I think not having functions around modal wrappers (if no extra logic is required) is also more readable.